source/docs/articles/sites/redis-as-a-caching-backend.md

```diff
@@ -106,6 +105,9 @@ The common community module for Drupal to use Redis is simply called [redis](htt
  ![](/source/docs/assets/images/desk_images/71423.png)
  - For Drupal 6 visit  `admin/settings/performance/cache-backend` and you should be able to see the available backends and their statuses.
 
+ ### Using Redis with Drupal 8 
+ At this time, sites running Drupal 8 cannot use Redis. Check the status on the [Redis project page](https://www.drupal.org/project/redis), or view to the [Port Redis issue queue](https://www.drupal.org/node/2233413)for updates.

```

Comment:
Can we add the why?
"...cannot use Redis, because there isn't a Drupal 8 release of the Redis module yet. Check the status..."